misc/update_authors.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh -ex
+set -e
+
+if [ -d .git ]; then
+  echo "Updating authors...\n"
+
+  # Full git history is needed for this to work
+  # (hide stderr and return code since this fails in case full history is already available)
+  git fetch --unshallow 2> /dev/null || true
```

Comment:
Let's call not committing a potential for future improvements, if that's ok :-)

CMake brings the ability to run make package and make source_package. It will allow generating source and binary packages with one single command. Updating authors is kind of a prerequisite for that ability.
I know it's only you and you can do it manually. But I like automating things :-)
In the other PR those packages are both generated by travis, extracted out of source and automatically tested. Fells like a big step towards simplifying releases, delivering weekly binaries, etc.

Even if it's just you, it should be simple :-)

Reply:
P.S. this is required for travis because it doesn't clone with full history.

Reply:
> Even if it's just you, it should be simple :-)

My reaction is only that adding unneeded complexity to the build and release scripts does not make things simpler _for me_, rather the opposite. If there is a need for it later, sure, it's just that it wasn't obvious from your changes alone.

> But I like automating things :-)

You make it sound like I don't like to automate things, which is incorrect. I already _have_ it simple and automated.